.buildkite/pipeline.yml

```diff
@@ -69,11 +69,6 @@ steps:
         agents:
           queue: mac
         command: |
-          echo "--- :macos: Install Rosetta to fix x64 xattr issue on arm64"
-          # Install Rosetta to try and address xattr architecture-based issue
-          # See https://buildkite.com/automattic/studio/builds/1543#01904ffb-8498-4d13-aa5a-e2b3c6dbb302
-          softwareupdate --install-rosetta --agree-to-license
-
```

Comment:
As the successful build for this commit shows, this step was unnecessary 

🧹 🧹 🧹 💨 💨 💨 

https://buildkite.com/automattic/studio/builds/1661#01905da0-4d67-43b3-ae0f-6f59c828887f